Src/BootCamp.Chapter/MakeStuffWithInput.cs

```diff
@@ -0,0 +1,242 @@
+            else
+            {
+                return people + " had the most money ever. ¤" + amount + ".";
+            }
+
+        }
+        public static string MakePadding(string message, int padding)
+        {
+            const string corner = "+";
+            const string horizontal = "-";
+            const string vertical = "|";
+
+            string[] splitmessage = message.Split(Environment.NewLine);
+            int height = splitmessage.Length + (padding * 2) + 1;
+            int width = 0;
+
+            foreach (string line in splitmessage) // checking which word is the longest to know how wide the padding should be.
+            {
+                if(line.Length > width)
+                {
+                    width = line.Length;
+                }
+            }
+            width += (padding * 2) + 1;
+            return MakeTabel();
+
+            string MakeTabel()
+            {
+                StringBuilder final = new StringBuilder();
+                for (int iHeight = 0; iHeight <= height; iHeight++)
+                {
+                    for (int kWidth = 0; kWidth <= width; kWidth++)
+                    {
+
+                        if (IsThisCorner(iHeight, kWidth))
+                        {
+                            final.Append(corner);
+                        }
+                        else if (IsThisHorizontal(iHeight, kWidth))
+                        {
+                            final.Append(horizontal);
+                        }
+                        else if (IsThisVertical(iHeight, kWidth))
+                        {
+                            final.Append(vertical);
+                        }
+                        else if (IsThisText(iHeight, kWidth))
+                        {
+                            final.Append(WhatLetterToBePlaced(iHeight, kWidth));
+                        }
+                        else if (IsThisWhiteSpace(iHeight, kWidth))
+                        {
+                            final.Append(" ");
+                        }
+                    }
+                    final.Append(Environment.NewLine);
+                }
+
+                return final.ToString();
+            }
+
+            bool IsThisCorner(int iHeight, int kWidth)
+            {
+                // corners = 0 + 0 =====  0 + width === height + 0 === height + width
+                if ((iHeight == 0 && kWidth == 0) || (iHeight == 0 && kWidth == width)||
+                    (iHeight == height && kWidth == 0)|| (iHeight == height && kWidth == width))
+                {
+                    return true;
+                }
+
+                return false;
+            }
+            bool IsThisHorizontal(int iHeight, int kWidth)
+            {
+                // horizontal = height 0 !& width 0 !& width maxwidth || height max !& width 0 !& width maxwidth
+                if ((iHeight == 0 && !(kWidth == 0 || kWidth == width)) || (iHeight == height && !(kWidth == 0 || kWidth == width)))
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Reduce the number of conditional operators (5) used in the expression (maximum allowed 3).](https://app.codacy.com/manual/Almantask/CSharp-From-Zero-To-Hero/pullRequest?prid=5123803)